.env-devel

```diff
@@ -87,6 +87,7 @@ TRACING_THRIFT_COMPACT_ENDPOINT=http://jaeger:5775
 TRAEFIK_SIMCORE_ZONE=internal_simcore_stack
 
 # NOTE: WEBSERVER_SESSION_SECRET_KEY = $(python3 -c "from cryptography.fernet import Fernet; print(Fernet.generate_key())")
+PROJECTS_MAX_COPY_SIZE_BYTES=32212254720
```

Comment:
Why not having human readable numbers and doing the conversion in code? e.g. ``PROJECTS_MAX_COPY_SIZE_GB``

Reply:
actually it **already** works like that. will change the .env-devel accordingly.